SDGupdater/R/unique_to_string.R

```diff
@@ -0,0 +1,23 @@
+#' Unique entries to string
+#'
+#' Turns a vector that hold only character strings into a single character
+#' string, with values separated by separated by ", ". NAs are ignored.
+#'
+#' @param variable Vector of character strings or numbers.
```

Comment:
Again, I can see this info in the function code, but doesn't show up for `?unique_to_string`, it still says "Vector of character strings" only. I think it's my RStudio playing up, but worth quick check to see if you're getting it as expected